libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -128,8 +128,9 @@ class Compiler {
 
   // Supported optimization levels.
   enum class OptimizationLevel {
-    Zero,  // No optimization.
-    Size,  // Optimization towards reducing code size.
+    Zero,         // No optimization.
+    Performance,  // Optimization towards better performance.
```

Comment:
Keep these in same order as the C header.    So reorder to match requested reorder there.

Reply:
Fixed.